test/fixtures/served/iframe.html

```diff
@@ -52,6 +52,12 @@
             sentinel: sentinel,
             type: 'send-embed-state',
           }, '*');
+    } else if (event.data.type === 'requestPymjsHeight') {
+      var msg = [ 'pym', 'height', event.data.height ].join( 'xPYMx' );
+      getAmpWindow(sentinel)./*OK*/postMessage( msg, '*');
+    } else if (event.data.type === 'requestPymjsWidth') {
+      var msg = [ 'pym', 'width', event.data.height ].join( 'xPYMx' );
```

Comment:
Ditto.

Reply:
Fixed in 2d578da50ec1cb3cb7626d98683aea9fba030568